release.config.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  branches: [{ name: 'release' }],
+  plugins: [
+    [
+      '@semantic-release/commit-analyzer',
+      {
+        preset: 'angular',
+        releaseRules: [
```

Comment:
This means that only `.PATCH` will be bumped for these types of commits. `feat` and `fix` will be completely ignored.

Reply:
As dicussed: should be added to the `README` because it deviates from the rest of the projects' setup.